frontend/src/metabase/admin/settings/components/Email/CloudSMTPConnectionForm.tsx

```diff
@@ -0,0 +1,293 @@
+
+  const handleUpdateEmailSettings = useCallback(
+    async (formData: FormValues) => {
+      const smtpPort = parseInt(
+        formData["cloud-email-smtp-port"],
+      ) as CloudEmailSMTPSettings["cloud-email-smtp-port"];
+
+      try {
+        await updateCloudEmailSMTPSettings({
+          ...formData,
+          "cloud-email-smtp-port": smtpPort,
+        }).unwrap();
+        trackSMTPSetupSuccess({ eventDetail: "cloud" });
+        sendToast({
+          message: t`Email settings updated`,
+        });
+        onClose();
+      } catch (error) {
+        sendToast({
+          icon: "warning",
+          toastColor: "error",
+          message: getErrorMessage(error, t`Error updating email settings`),
+        });
+
+        // throw to allow the form to handle the error
+        throw error;
+      }
+    },
+    [updateCloudEmailSMTPSettings, sendToast, onClose],
+  );
+
+  const allSetByEnvVars = useMemo(() => {
+    return (
+      settingsDetails &&
+      emailSettingKeys.every(
+        (settingKey) => settingsDetails[settingKey]?.is_env_setting,
+      )
+    );
+  }, [settingsDetails]);
+
+  return (
+    <Modal
+      title={t`SMTP Configuration`}
+      opened
+      onClose={onClose}
+      padding="xl"
+      data-testid="cloud-smtp-connection-form"
+    >
+      <Box data-testid="settings-updates" pt="lg">
+        <FormProvider
+          initialValues={initialValues}
+          validationSchema={getFormValueSchema(settingsDetails)}
+          onSubmit={handleUpdateEmailSettings}
+          enableReinitialize
+        >
+          {({ dirty, isValid, isSubmitting }) => (
+            <Form>
+              <Stack gap="lg">
+                <SetByEnvVarWrapper
+                  settingKey="cloud-email-smtp-host"
+                  settingDetails={settingsDetails?.["cloud-email-smtp-host"]}
+                >
+                  <FormTextInput
+                    name="cloud-email-smtp-host"
+                    label={t`SMTP Host`}
+                    description={
+                      settingsDetails?.["cloud-email-smtp-host"]?.description
+                    }
+                    placeholder={"smtp.yourservice.com"}
+                  />
+                </SetByEnvVarWrapper>
+                <SetByEnvVarWrapper
+                  settingKey="cloud-email-smtp-port"
+                  settingDetails={settingsDetails?.["cloud-email-smtp-port"]}
+                >
+                  <FormChipGroup
+                    name="cloud-email-smtp-port"
+                    label={t`SMTP Port`}
+                    groupProps={{ mt: "0.5rem" }}
+                  >
+                    <Chip value={"465"} variant="brand">
+                      465
+                    </Chip>
+                    <Chip value={"587"} variant="brand">
+                      587
+                    </Chip>
+
+                    <Chip value={"2525"} variant="brand">
+                      2525
+                    </Chip>
```

Comment:
why don't we support arbitrary ports, like on self-hosted?

Reply:
It's a requirement that they set up a secure SMTP server, so we limit it to secure ports and security is required (in self-hosted there's a 'none' option. @nvoxland can probably explain it better than me but it's to do with our hosted AWS setup that we can't just connect to random SMTP servers (maybe SPAM related)

Reply:
I am not an expert on this, so @nvoxland or @dpsutton please chime in here, but I believe these are the standard secure SMTP ports, and providers such as AWS don't like making connections to non-standard ports because spam